src/Identity/Controllers/AccountsController.cs

```diff
@@ -21,27 +27,38 @@ namespace Bit.Identity.Controllers;
 [ExceptionHandlerFilter]
 public class AccountsController : Controller
 {
+    private readonly ICurrentContext _currentContext;
     private readonly ILogger<AccountsController> _logger;
     private readonly IUserRepository _userRepository;
     private readonly IUserService _userService;
     private readonly ICaptchaValidationService _captchaValidationService;
     private readonly IDataProtectorTokenFactory<WebAuthnLoginAssertionOptionsTokenable> _assertionOptionsDataProtector;
     private readonly IGetWebAuthnLoginCredentialAssertionOptionsCommand _getWebAuthnLoginCredentialAssertionOptionsCommand;
+    private readonly ISendVerificationEmailForRegistrationCommand _sendVerificationEmailForRegistrationCommand;
+    private readonly IReferenceEventService _referenceEventService;
+
 
     public AccountsController(
+        ICurrentContext currentContext,
         ILogger<AccountsController> logger,
         IUserRepository userRepository,
         IUserService userService,
         ICaptchaValidationService captchaValidationService,
         IDataProtectorTokenFactory<WebAuthnLoginAssertionOptionsTokenable> assertionOptionsDataProtector,
-        IGetWebAuthnLoginCredentialAssertionOptionsCommand getWebAuthnLoginCredentialAssertionOptionsCommand)
+        IGetWebAuthnLoginCredentialAssertionOptionsCommand getWebAuthnLoginCredentialAssertionOptionsCommand,
+            ISendVerificationEmailForRegistrationCommand sendVerificationEmailForRegistrationCommand,
+            IReferenceEventService referenceEventService
+        )
```

Comment:
nit: IDE probably did some weird tabbing to you here

Reply:
Hmm... weird. I fixed it!